android/src/main/kotlin/com/xraph/plugin/flutter_unity_widget/UnityPlayerUtils.kt

```diff
@@ -43,6 +46,27 @@ class UnityPlayerUtils {
                     if (!reInitialize) {
                         activity.window.setFormat(PixelFormat.RGBA_8888)
                         unityPlayer = UnityPlayer(activity, ule)
+
+                        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.O) {
```

Comment:
I don't really understand any of this lol. Which is fine. Just one question out of curiosity. This if seems to be checking if the version number is below zero? Wondering what this is checking.

Reply:
`Build.VERSION_CODES.O` at first glance does look like a 0. I wish they didn't truncate "Oreo" to "O". At runtime this constant int is 26. https://developer.android.com/reference/android/os/Build.VERSION_CODES#O

Reply:
Regarding what the code does, the majority of the work is happening within the unity generated library via `unityPlayer!!.addViewToPlayer(view, true)`. The code above that is instantiating a `TextureView` from the Android SDK and setting it up so that the unity player view renders to it. The unity player shows our newly made `TextureView` because calling `addViewToPlayer` swaps the internal surface with our `TextureView`.